components/layouts/partials/navigation.js

```diff
@@ -26,6 +28,8 @@ const List = styled.ul`
 
 export default function Navigation({ closeNavigation }) {
     const auth = useAuth();
+    const matrix = useMatrix(auth.getAuthenticationProvider('matrix'));
+    const invites = matrix.invites;
```

Comment:
Is this variable necessary? Or can we just say `matrix.invites.size` down there?